stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -31,3 +32,29 @@ logs = (start, stop=now(), fn) =>
 deadman = (t, tables=<-) => tables
     |> max(column: "_time")
     |> map(fn: (r) => ( {r with dead: r._time < t} ))
+
+// levels describing the result of a check
+levelOK = 0
+levelInfo = 1
+levelWarn = 2
+levelCrit = 3
+levelUnknown = -1
+
+// Check performs a check against its input using the given ok, info, warn and crit functions
+// and writes the result to a system bucket.
+check = (tables=<-, data, messageFn, crit, warn, info, ok) =>
```

Comment:
`data` should be optional I think. So it would default to `data={}` if that's possible.

Reply:
If it's optional then the `_check_id` would be null, and then writing it to storage would... just omit the tag?  actually I'm not sure what would happen.

Reply:
I figured these were more generic functions and that the actual influxdb implementation would invoke the function correctly.